packages/aws-cdk-lib/aws-kms/test/key.test.ts

```diff
@@ -518,12 +520,30 @@ test('key with some options', () => {
   });
 });
 
+test('set rotationPeriod without enabling enableKeyRotation', () => {
```

Comment:
Can we also add a test with `enableKeyRotation` explicitly set to true? Other than that this looks ready to ship.

Reply:
you are correct, it does.